layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigParser.java

```diff
@@ -24,6 +24,19 @@
  */
 public interface LayersConfigParser {
 
+    /**
+     * Returns the set of supported mime-types by this parser.
+     * Used to determine if the content of a given URL can be read by this parser.
+     * @return the set of supported mime-types by this parser, should never return {@code null}.
+     */
+    String[] getSupportedMimeTypes();
```

Comment:
Can we make it a `Set<String>` instead of an array? Expresses the intended identity.

Reply:
Sure, we can switch to `Set<String>`